addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -159,6 +185,22 @@ function factory(dependencies) {
                 this.threadDescriptionInputRef.el.focus();
                 this.update({ doFocusOnThreadDescriptionInput: clear() });
             }
+            if (
+                this.doSetSelectionStartOnGuestNameInput !== undefined &&
+                this.doSetSelectionEndOnGuestNameInput !== undefined &&
+                this.doSetSelectionDirectionOnGuestNameInput !== undefined
+            ) {
+                this.guestNameInputRef.el.setSelectionRange(
+                    this.doSetSelectionStartOnGuestNameInput,
+                    this.doSetSelectionEndOnGuestNameInput,
+                    this.doSetSelectionDirectionOnGuestNameInput
+                );
+                this.update({
+                    doSetSelectionDirectionOnGuestNameInput: clear(),
+                    doSetSelectionEndOnGuestNameInput: clear(),
+                    doSetSelectionStartOnGuestNameInput: clear(),
+                });
+            }
```

Comment:
A lot of repetition again for this, we will need to factorize this kind of code at some point (not now).